ci-operator/config/openshift/release-tests/openshift-release-tests-master.yaml

```diff
@@ -0,0 +1,31 @@
+build_root:
+  image_stream_tag:
+    name: "4.14"
+    namespace: ocp
+    tag: root
+images:
+- dockerfile_path: prow/Dockerfile
+  from: root
+  to: job-python3
+promotion:
+  namespace: ci
+  tag: master
+  tag_by_commit: true
```

Comment:
@jianzhangbjz it is not used. we do not need `job-python3:<commit-hash>`, and just need `job-python3:master`

Reply:
ok, updated it.